### DISCUSSION TITLE
I am not able to use Logger component in Aura component

### DISCUSSION THREAD

**Discussion by kalpeshvyas26** [jongpie/NebulaLogger#599] [Q&A] [answered]
I installed the managed package and tried to use this in custom Aura component but getting error.

![image](https://github.com/jongpie/NebulaLogger/assets/29892999/4425d16d-0578-407f-a97b-c848e48f0963)

Any idea what is the problem?

**Comment by jongpie** [comment]
Hi @kalpeshvyas26 - unfortunately, the managed package currently does not support logging in Aura or LWC. There are a few open issues with JavaScript logging that I'd like to get fixed before I add the functionality in the managed package - you can read some of [my comments in issue 533](https://github.com/jongpie/NebulaLogger/issues/533#issuecomment-1803057436). I'm hoping to resolve some of these issues in the next few months.

**Comment by kalpeshvyas26** [comment]
Thank you for your response Jonathan.
Can you let me know when it can be available in managed package?

**Reply by jongpie** [reply]
Sure, I'll do my best to remember to comment here again once it's ready 👍

**Reply by jongpie** [reply, marked as answer]
@kalpeshvyas26 the `logger` LWC is now available in the managed package in [release `v4.13.0`](https://github.com/jongpie/NebulaLogger/releases/tag/v4.13.0)!

**Comment by kalpeshvyas26** [comment]
Awesome, thanks! For letting me know
